src/app/credExplorer/repoRegistry.js

```diff
@@ -0,0 +1,8 @@
+// @flow
+
+// The repoRegistry is written by the CLI load command
+// (src/cli/commands/load.js) and is read by the RepositorySelect component
+// (src/app/credExplorer/RepositorySelect.js)
+export const REPO_REGISTRY_FILE = "repositoryRegistry.json";
+export const REPO_REGISTRY_COMPAT = {type: "REPO_REGISTRY", version: "0.1.0"};
```

Comment:
Two problems with this:
 1. By requiring clients to call `fromCompat` themselves, but only
    providing the most recent compat information, we prevent clients
    from using the backward compatibility feature of `compat.js`.
 2. With the current typings, a client can overwrite the attributes of
    this object, polluting other clients.

Why not treat this module as a provider of an actual data type? You can
`export type RepoRegistry`, and provide `toCompat` and `fromCompat`. You
could additionally export `emptyRegistry: RepoRegistry`, if you want.
Then, the `REPO_REGISTRY_COMPAT` is an implementation detail.

The two `_FILE` and `_API` strings can continue to exist.